ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -1018,3 +1035,20 @@ func compareRayClusterJsonHash(spec1 rayv1alpha1.RayClusterSpec, spec2 rayv1alph
 	}
 	return hash1 == hash2, nil
 }
+
+// getHeadPod returns the head pod of the RayCluster.
```

Comment:
what is `getHeadPod`? is it used anywhere?

Reply:
Good catch! I forgot to update the comments. At first, I wanted to implement `getHeadPod` instead of `isHeadPodRunningAndReady`. I will update it.

Reply:
Updated: https://github.com/ray-project/kuberay/pull/1074/commits/653681170e34060af4649b3f8352c7f6fad9291d